### DISCUSSION TITLE
Fabric?

### DISCUSSION THREAD

**Discussion by cutter4ever** [ldtteam/minecolonies#7957] [Ideas]
Just wondering If yall will be doing a Fabric version for 1.18 or not. Tks

**Comment by Raycoms** [comment]
No. Way too much work to maintain and port.

**Reply by RedstoneWizard08** [reply]
(Sorry for reviving a dead thread but...) architectury? Porting lib?

**Reply by Thodor12** [reply]
Even then it's not feasible, it's a complete rewrite from the ground up

**Reply by RedstoneWizard08** [reply]
Fair
